e2e-pw/src/oss/specs/color-scheme/color-scheme.spec.ts

```diff
@@ -0,0 +1,221 @@
+      { animations: "allow" }
+    );
+  });
+
+  test("should be able to update color by value - tag", async ({
+    gridActionsRow,
+    colorModal,
+    page,
+    eventUtils,
+    sidebar,
+  }) => {
+    // turn on the sample tag bubble
+    await sidebar.clickFieldCheckbox("tags");
+    // mount eventListener
+    const gridRefreshedEventPromise =
+      eventUtils.getEventReceivedPromiseForPredicate(
+        "re-render-tag",
+        () => true
+      );
+    // open color modal and modify color in sample tags field and ground_truth
+    await gridActionsRow.toggleColorSettings();
+
+    await colorModal.selectActiveField("sample tags");
+    await colorModal.toggleColorMode();
+
+    await page
+      .getByTitle(`Use custom colors for specific field values`)
+      .first()
+      .click({ force: true });
+    await colorModal.addANewPair("validation", "yellowgreen");
+    await colorModal.addANewPair("test", "black"); // this doesn't matter
+
+    await colorModal.closeColorModal();
+    const tagBubble = page.getByTestId("tag-validation").first();
+
+    await gridRefreshedEventPromise;
+    await gridRefreshedEventPromise;
+    // verify validation tag has yellow green as background color
+    expect(await tagBubble.getAttribute("style")).toContain(
+      "rgb(154, 205, 50)"
+    );
+  });
+
+  test("should be able to update color by value - detection - no colorbyattribute", async ({
+    grid,
+    gridActionsRow,
+    colorModal,
+    page,
+    sidebar,
+  }) => {
+    // turn off predictions detections and open color modal
+    await sidebar.clickFieldCheckbox("predictions");
+    await gridActionsRow.toggleColorSettings();
+    // go to prediction tab
+    await colorModal.selectActiveField("ground_truth");
+    // color by value mode
+    await colorModal.toggleColorMode();
+
+    await page
+      .getByTitle(`Use custom colors for specific field values`)
+      .first()
+      .click({ force: true });
+    // I can set value colors directly bypass choosing attribute by value
+    await colorModal.addANewPair("bird", "yellow");
+    await colorModal.addANewPair("person", "red");
+    await colorModal.addANewPair("horse", "green");
+    await colorModal.addANewPair("cat", "blue");
+    await colorModal.addANewPair("bottle", "white");
+    await colorModal.addANewPair("surfboard", "white");
+    await colorModal.addANewPair("knife", "white");
+    await colorModal.addANewPair("fork", "white");
+    await colorModal.addANewPair("cup", "white");
+    await colorModal.addANewPair("dining table", "white");
+    await colorModal.addANewPair("chair", "white");
+    await colorModal.addANewPair("cake", "white");
+
+    await colorModal.closeColorModal();
+    await expect(await grid.getNthFlashlightSection(0)).toHaveScreenshot(
+      "color-value-setting-1.png",
+      { animations: "allow" }
+    );
+  });
+
+  test("should be able to update color by value - ground_truth - with custom string field", async ({
```

Comment:
super nit: we could drop all the `should be able to` prefix to test descriptions probably
for example: here would be `update by ground_truth value and custom string field works`